site/content/en/docs/manual/advanced/analytics-and-monitoring/consensus.md

```diff
@@ -0,0 +1,179 @@
+- replicas are not included in task annotation import or export.
+  Per-job import and export still works for any job types
+- replicas use the same frame range as their parent annotation jobs
+- only annotation jobs can have replicas
+
+Read more about merging [here](#how-to-merge-all-replicas-in-a-task).
+
+## Workflow
+
+When annotating with consensus, the typical workflow looks this way:
+
+1. Create a task with consensus enabled. Optionally,
+  {{< ilink "/docs/manual/basics/create_an_annotation_task" "configure validation" >}}
+2. Assign annotators to consensus replicas, wait until all the jobs are completed
+3. Once all replicas in a parent job are completed, merge annotations
+4. Review and resolve problems in the parent jobs
+
+### How to enable consensus in a task
+
+Consensus annotation is configured on the Task level. It is only available at task creation.
+If you want to enable consensus for one of your existing tasks, you'll need to recreate the task.
+
+1. Go to the {{< ilink "/docs/manual/basics/create_an_annotation_task" "task creation" >}} page
+2. Configure basic and advanced parameters according to your requirements,
+  and attach a dataset to be annotated.
+3. To enable consensus for the task, open the **Advanced** section and
+  set **Consensus Replicas** to above 1.
+
+  ![Consensus replicas parameter image](/images/consensus-replicas-task-parameter.jpg)
+
+4. Create the task and open the task page
+
+If a task has consensus enabled, you'll see **Consensus** tag in the task summary.
+Existing **Consensus replica** jobs will be displayed in the jobs list under their parent
+annotation jobs.
+
+  ![Consensus replica jobs in the task job list](/images/consensus-replicas-list-task.jpg)
+
+### Merging
+
+For a given parent task with related annotated consensus jobs, merging will
+match annotations between replicas and save merged annotations into the parent job.
+
+> Please note that the **merging overrides annotations in the parent job**.
+> This operation cannot be undone. Please make sure that the parent job
+> is ready for merging and backup annotations if needed.
+
+There are 2 merging options available:
+- merge replicas in all available parent jobs in a task
+- merge replicas in a specific parent job
+
+Merging is only available for a parent job if it is on the **annotation** stage and
+it has at least 1 replica not in the **annotation** stage and **new** state.
+For simplicity, this can be read as "if there are any annotated replicas in the parent job".
+
+After merging, parent jobs are switched to the **completed** state automatically.
+If you prefer using merging at the task level, it is recommended to switch merged
+parent jobs to the **validation** stage after they are merged to exclude them from
+the next merging and avoid losing the reviewed annotations.
+
+### How to merge all replicas in a task
+
+1. Open the task **Actions** menu
+2. Click **Merge consensus jobs**
```

Comment:
```suggestion
1. Click **Merge consensus jobs**
```

Etc. Just to prepare for future editing.

Reply:
Unfortunately, it doesn't work in complex cases, including this one.